src/Entity/Adherent.php

```diff
@@ -2453,6 +2455,50 @@ public function isCandidate(): bool
         return $this->candidateManagedArea instanceof CandidateManagedArea;
     }
 
+    public function isDelegatedCandidate(): bool
+    {
+        return \count($this->getReceivedDelegatedAccessOfType('candidate')) > 0;
+    }
+
+    public function isDelegatedHeadedRegionalCandidate(): bool
+    {
+        foreach ($this->receivedDelegatedAccesses as $delegatedAccess) {
```

Comment:
could we use `$this->getReceivedDelegatedAccessOfType('candidate')` and avoid `'candidate' === $delegatedAccess->getType()` check?

```php
foreach ($this->getReceivedDelegatedAccessOfType('candidate') as $delegatedAccess)
```